proto/backend.proto

```diff
@@ -134,7 +136,8 @@ message CheckHealth {
     }
 
     HealthStatus status = 1;
-    string info = 2;
+    string message = 2;
```

Comment:
breaking change but I thought this is more clear.

Reply:
👍